.github/workflows/slugify-value.yaml

```diff
@@ -57,10 +57,10 @@ jobs:
       - name: Validate // Slugify key/value
         run: |
           [[ "${{ env.KEY_VALUE_TEST }}" == "refs/pulls/feat/-----Some----Changes_to.be------" ]]
```

Comment:
I think we can change the test value to be more useful on each test.
So you can use another value that fit the test case better.